Source/santasyncservice/syncv1.proto

```diff
@@ -197,6 +199,8 @@ message Event {
 
 message EventUploadRequest {
   repeated Event events = 1;
+  // The UUID of the machine that generated this event.
```

Comment:
nit: There could be multiple events. Maybe the comment can be `The UUID of the machine making the request` for all of the request types?

Reply:
What about `The UUID of the machine where the event(s) occurred`?

Reply:
That works too